numpy/random/mtrand/randomkit.c

```diff
@@ -240,6 +246,101 @@ rk_random(rk_state *state)
     return y;
 }
 
+
+/*
+ * Returns an unsigned 64 bit random integer.
+ */
+NPY_INLINE static npy_uint64
+rk_uint64(rk_state *state)
+{
+    npy_uint64 upper = (npy_uint64)rk_random(state) << 32;
+    npy_uint64 lower = (npy_uint64)rk_random(state);
```

Comment:
If we know they are zero it is ok. I was worried they might have junk bits which could interfere with the distribution and the determinism.